java/src/main/java/com/company/automation/classes/main_task/StudentLogic.java

```diff
@@ -0,0 +1,100 @@
+package com.company.automation.classes.main_task;
+
+public class StudentLogic {
+    private final static StudentLogic studentLogic = new StudentLogic();
+
+    private StudentLogic() {
+    }
+
+    public static StudentLogic getStudentLogic() {
+        return studentLogic;
+    }
+
+    public String getStudentFullInfo(Student student) {
+        return "Id: " + student.getId() + ". " + student.getFullName() + ". Birth date: " + student.getBirthDate() +
+                "\nAddress: " + student.getAddress() + "\nPhone number: " + student.getPhoneNumber() + "\n" +
+                student.getFaculty().getName() + "\n" + student.getCourse().getNumber() + " course. Group: " + student.getGroup() + "\n";
+    }
+
+    public Student[] getStudentsOfGroup(Student[] students, int group) {
```

Comment:
I think it should have a name like getSortedStudentsOfGroup

Reply:
fixed